Setup/UpgradeData.php

```diff
@@ -0,0 +1,140 @@
+                    ['config_id', 'path', 'value']
+                )->where(
+                    '`path` = "myparcelbe_magento_standard/options/insurance_500_active"'
+                );
+
+                $ShippingMethodData = $connection->fetchAll($selectShippingMethodSettings);
+                if ($ShippingMethodData) {
+                    foreach ($ShippingMethodData as $value) {
+                        $fullPath = 'myparcelbe_magento_bpost_settings/default_options/insurance_500_active';
+                        $bind     = ['path' => $fullPath, 'value' => $value['value']];
+                        $where    = 'config_id = ' . $value['config_id'];
+                        $connection->update($table, $bind, $where);
+                    }
+                }
+                /*
+                * Move signature_active to carrier settings
+                */
+                $selectShippingMethodSettings = $connection->select()->from(
+                    $table,
+                    ['config_id', 'path', 'value']
+                )->where(
+                    '`path` = "myparcelbe_magento_standard/options/signature_active"'
+                );
+
+                $ShippingMethodData = $connection->fetchAll($selectShippingMethodSettings);
+                if ($ShippingMethodData) {
+                    foreach ($ShippingMethodData as $value) {
+                        $fullPath = 'myparcelbe_magento_bpost_settings/default_options/signature_active';
+                        $bind     = ['path' => $fullPath, 'value' => $value['value']];
+                        $where    = 'config_id = ' . $value['config_id'];
+                        $connection->update($table, $bind, $where);
+                    }
+                }
+                /*
+                 * Move myparcelbe_magento_checkout to myparcelbe_magento_bpost_settings
+                 */
+                $selectCheckoutSettings = $connection->select()->from(
+                    $table,
+                    ['config_id', 'path', 'value']
+                )->where(
+                    '`path` LIKE "myparcelbe_magento_checkout/%"'
+                );
+
+                $checkoutData = $connection->fetchAll($selectCheckoutSettings);
+                if ($checkoutData) {
+                    foreach ($checkoutData as $value) {
+                        $path    = $value['path'];
+                        $path    = explode("/", $path);
+                        $path[0] = 'myparcelbe_magento_bpost_settings';
+
+                        $fullPath = implode("/", $path);
+
+                        $bind  = ['path' => $fullPath, 'value' => $value['value']];
+                        $where = 'config_id = ' . $value['config_id'];
+                        $connection->update($table, $bind, $where);
+                    }
+                }
+
+                /*
+                 * Insert bpost enabled data
+                 */
+                if ($connection->isTableExists($table) == true) {
```

Comment:
Op regel 39 check je al of die tabel bestaat.

Reply:
Die was ik inderdaad vergeten weg te halen: Done